Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -191,6 +191,45 @@ def parse_mail_parts(parts):
     return body, html, attachments
 
 
+def format_fields_argument(fields: list[str]) -> list[str] | None:
+
+    full_fields = [
+        "Type",
+        "Mailbox",
+        "ThreadId",
+        "Labels",
+        "Headers",
+        "Attachments",
+        "RawData",
+        "Format",
+        "Subject",
+        "From",
+        "To",
+        "Body",
+        "Cc",
+        "Bcc",
+        "Date",
+        "Html",
+        "Attachment Names",
+    ]
+    if valid_fields := list(
+        filter(
+            lambda field: field.lower() in [field.lower() for field in fields],
+            full_fields,
+        )
+    ):
```

Comment:
```suggestion
    valid_fields = (
        "Type",
        "Mailbox",
        "ThreadId",
        "Labels",
        "Headers",
        "Attachments",
        "RawData",
        "Format",
        "Subject",
        "From",
        "To",
        "Body",
        "Cc",
        "Bcc",
        "Date",
        "Html",
        "Attachment Names",
    )
    
    lower_valid_fields = {field.lower() for field in fields}
    if valid_fields := [field for field in fields if field in lower_valid_fields]:
```
1. better search in a set than in a list
2. more readable (set has a name)